### PR TITLE
Replace chrome.i18n with angular gettext

### LINKED ISSUE #1: Replace chrome.i18n with angular-gettext

### PR DESCRIPTION
**Pull request by tlvince**
Closes #1.

### CODE REVIEW

**Comment by tlvince** [comment]
@karlwestin sorry its huge :scream:. Cursory run through of the app suggests nothing is outwardly broken.

**Comment by karlwestin** [comment]
Yeah but that's fine :) im trying to test it but i can't access the UOM db

File: app/views/stock-count/stock-count-form.html [hunk 1]
```diff
@@ -237,8 +237,8 @@ <h3 class="center-text bold-text" ng-show="countValue[productKey] >= 0">
                             </div>
                         <div class="btn-group">
                           <button ng-click="finalSave()" class="btn btn-success btn-lg" ng-disabled="isSaving">
-                            <span i18n="confirm" ng-show="!isSaving"></span>
-                            <span ng-show="isSaving">Saving</span>
+                            <span translate ng-show="!isSaving">Confirm</span>
+                            <span ng-show="isSaving" translate>Saving</span>
```
Comment (karlwestin) [inline app/views/stock-count/stock-count-form.html]: ngShow + translate

**Comment by tlvince** [comment]
@karlwestin what's the issue with `translate` on the same element as `ng-show`/`ng-if`? I tested adding a translation (en_GB) and it looks fine:

screen shot 2015-06-05 at 12 55 20
screen shot 2015-06-05 at 12 49 44
screen shot 2015-06-05 at 12 54 47

**Comment by karlwestin** [comment]
@tlvince let's leave the ng-ifs etc in then, maybe the bug has been fixed :)
lets just add a comment in the messages file, and then we're good to go!

**Comment by karlwestin** [comment]
Yay! High five!

**Pull request merged**